src/main/ts/rules/PreferFun.ts

```diff
@@ -40,46 +94,60 @@ export const preferFun: Rule.RuleModule = {
     messages: {
       preferNoop: 'Use `Fun.noop` instead of redeclaring a no-op function, eg: `() => {}`',
       preferAlways: 'Use `Fun.always` instead of redeclaring a function that always returns true, eg: `() => true`',
-      preferNever: 'Use `Fun.never` instead of redeclaring a function that always returns false, eg: `() => false`'
-    }
+      preferNever: 'Use `Fun.never` instead of redeclaring a function that always returns false, eg: `() => false`',
+      preferConstant: 'Use `Fun.constant` instead of redeclaring a function that always returns the same value, eg: `() => 0`'
+    },
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          noop: { type: 'boolean' },
+          always: { type: 'boolean' },
+          never: { type: 'boolean' },
+          constant: { type: 'boolean' }
+        },
+        additionalProperties: false
+      }
+    ]
   },
   create: (context) => {
-    const reportIfRequired = (func: CallExpression | FunctionExpression | ArrowFunctionExpression, literal: Literal | null) => {
-      if (literal === null) {
-        context.report({
-          node: func,
-          messageId: 'preferNoop'
-        });
-      } else if (literal.raw === 'true') {
-        context.report({
-          node: func,
-          messageId: 'preferAlways'
-        });
-      } else if (literal.raw === 'false') {
+    const options = context.options[0] || {};
+
+    const report = (func: CallExpression | FuncExpression, type: Option) => {
+      if (options[type.toLowerCase()] !== false) {
         context.report({
           node: func,
-          messageId: 'preferNever'
+          messageId: `prefer${type}`
```

Comment:
Why not just add the prefer to the string values of the enum?

Reply:
It wouldn't match the option names then...